tests/fsharpqa/Source/Conformance/SpecialAttributesAndTypes/Imported/CallerInfo/CallerMemberName.fs

```diff
@@ -45,6 +47,8 @@ type MyTy() =
             |> List.head
         MyTy.Check(result, Some("CheckMembers"), "lambda")
         MyTy.Check(functionVal (), Some("functionVal"), "functionVal")
+        MyTy.Check(anonymousLambda, Some("anonymousLambda"), "anonymousLambda")
+        MyTy.Check(asyncVal, Some("asyncVal"), "asyncVal")
```

Comment:
@dsyme  I changed the tests in the previous one the asyncVal was member of MyTy and because of that the value was "asyncVa" now the asyncVal is value of main and the expected value is "main"